### DISCUSSION TITLE
Master branch appears twice in select branch

### DISCUSSION THREAD

**Discussion by benedettoraviotta** [mia-platform/community#73] [Q&A]
Hi, 

I encountered a graphical bug while changing a branch from the dedicated console section,  the `master` branch appears twice, as shown below:
![image](https://user-images.githubusercontent.com/59974565/200355089-7b6d37ac-a11e-4796-b05f-77347d5098f9.png)

From the network inspect, the request that is made to the console is shown below:
![network-bug-request](https://user-images.githubusercontent.com/59974565/200359909-eca1ff0e-0bf3-4b68-9eb9-821196feb223.jpg)

And the response, to the request effected above, is shown below::
![network-bug](https://user-images.githubusercontent.com/59974565/200359734-eeadde53-39a5-4214-8e82-9178807e610a.jpg)

The console version is v9.5.2. 
The git provider is Atlassian Bitbucket v7.21.0.

**Comment by fredmaggiowski** [comment]
Hi, thank you, I see the branch is appearing twice in the API response too so I think this may not be a UI error but rather an error in the backend API providing the branches 🤔 

We have to take a look into this, @albertotessarotto this seems like a bug to me
